### DISCUSSION TITLE
Data type list or "Repeatable Data Types"

### DISCUSSION THREAD

**Discussion by skttl** [leekelleher/umbraco-contentment#336] [Idea]
The use case for this is to be able to do a list of any datatype eg. Rtes. Think of repeatable textstrings but for any datatype. 

The return value would be IEnumerable<T> where T is the value type of the selected datatype.

**Comment by leekelleher** [comment]
For reference, I did similar for Umbraco 7 called Property List: https://github.com/umco/umbraco-property-list

**Comment by leekelleher** [comment]
Closing this off, as the feature will be part of **Input List** for Contentment v7.0. #513
